deployment/cloud-run/test_routing_debug.py

```
@@ -5,80 +5,153 @@
+            print("\n=== After adding namespace route ===")
+            print("App routes:", [rule.rule for rule in self.app.url_map.iter_rules()])
+
+            # Test direct Flask route
+            @self.app.route('/test')
+            def test():
+                return jsonify({'message': 'Test route'})
+
+            print("\n=== After adding Flask route ===")
+            print("App routes:", [rule.rule for rule in self.app.url_map.iter_rules()])
+
+    def test_routing_58(self):
```

Ruff (0.12.2)

Missing docstring in public method

(D102)